### PR TITLE
Master yog

### PR DESCRIPTION
**Pull request by Yog**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,8 @@
 
 [![Join the chat at https://gitter.im/stxnext-csr/volontulo](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/stxnext-csr/volontulo?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 Portal łączący wolontariuszy, organizacje i instytucje.
```
Comment (magul) [inline README.md]: If You're rewriting it in English, remove Polish version, please.

File: README.md [hunk 2]
```diff
@@ -2,3 +2,8 @@
...
+Web portal for collaboration of community volunteers with organizations and institutions
+
+## Instalation
+
+TODO
```
Comment (magul) [inline README.md:9]: There is not so much additional value in this TODO. Have You already installed app?

Comment (Yog) [inline README.md:9]: will do that later and then the instructuion shall come

File: README.md (hunk 1 above)
Comment (Yog) [inline README.md]: yup, my bad

**Pull request merged**